spk/squidguard/src/installer.sh

```diff
@@ -37,28 +37,40 @@ postinst ()
     # Patch template files
     hostname=`hostname`
     sed "s/==HOSTNAME==/$hostname/g" ${ETC_DIR}/squidguard.conf.tpl > ${ETC_DIR}/squidguard.conf
-    
+    sed "s/==HOSTNAME==/$hostname/g" ${ETC_DIR}/squidclamav.conf.tpl > ${ETC_DIR}/squidclamav.conf    
     # Correct the files ownership
     chown -R ${RUNAS}:users ${SYNOPKG_PKGDEST}
-    chown 0:0 ${SQUID_WRAPPER}
-	chmod +s ${SQUID_WRAPPER}
-	
-    # Init squid cache directory
-    su - ${RUNAS} -c "${SQUID} -z -f ${CFG_FILE}"
+
+    if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ]; then
+        # Init squid cache directory
+        su - ${RUNAS} -c "${SQUID} -z -f ${CFG_FILE}"
+    fi
+
+    # Init SSLBump cache directory
+    su - ${RUNAS} -c "${INSTALL_DIR}/libexec/ssl_crtd -c -s ${INSTALL_DIR}/var/ssl_db" >> /dev/null
 
     # Install webman
     ln -s ${WWW_DIR} ${WEBMAN_DIR}/${PACKAGE}
     ln -sf ${INSTALL_DIR}/etc/squidguardmgr.conf ${WEBMAN_DIR}/${PACKAGE}/
-      
-    # Init crontab : update squidguard DB each day at 1 a.m
+
+    # For squidclamav redirect
+    ln -sf ${INSTALL_DIR}/libexec/squidclamav/clwarn.cgi ${WEBMAN_DIR}/${PACKAGE}/clwarn.cgi
+
+    # Init crontab :
+    #    update squidguard DB each day at 1 a.m
+    #    logs rotate every sunday at 0:15 a.a
     grep ${PACKAGE} /etc/crontab
     if [ $? -eq 1 ]; then
-        echo "0 1       *       *       *       root    ${INSTALL_DIR}/bin/update_db.sh > ${INSTALL_DIR}/var/logs/update_db.log" >> /etc/crontab
-        /usr/syno/etc/rc.d/S04crond.sh stop
+        echo "0	1	*	*	*	root	${INSTALL_DIR}/bin/update_db.sh > ${INSTALL_DIR}/var/logs/update_db.log" >> /etc/crontab
+        echo "15	0	*	*	0	root	${INSTALL_DIR}/bin/logrotate.sh > ${INSTALL_DIR}/bin/logrotate.log" >> /etc/crontab
+        /usr/syno/etc/rc.d/S04crond.sh stop >> /dev/null
         sleep 1
-        /usr/syno/etc/rc.d/S04crond.sh start
+        /usr/syno/etc/rc.d/S04crond.sh start >> /dev/null
```

Comment:
This command is not compatible with DSM5.0 and higher. Might make sense to add cron to `INSTUNINST_RESTART_SERVICES`.

Also, please recheck if this actually works as you intend it to after rebooting your device. You might find that the extra entries you made manually will have disappeared.